Interop/StubGenerator/build.gradle

```diff
@@ -31,7 +31,7 @@ repositories {
 
 dependencies {
     compile "org.jetbrains.kotlin:kotlin-stdlib:$buildKotlinVersion"
-    compile "org.jetbrains.kotlin:kotlin-compiler:$buildKotlinVersion"
+    compile "org.jetbrains.kotlin:kotlin-compiler:$kotlinVersion"
```

Comment:
Why so?

Reply:
Because using `buildKotlinVersion` here was incorrect. `kotlinVersion` is a version of Kotlin frontend K/N is built against. `buildKotlinVersion` is our "bootstrap" version - version of the compiler K/N is built by.